moabb/tests/test_pipelines/CSP.yml

```diff
@@ -13,3 +13,5 @@ pipeline:
     from: sklearn.discriminant_analysis
     parameters:
       solver: svd
+
+param_grid: None
```

Comment:
Could you add some param_grid to test (nfilters or other), as test_pipelines are used in benchmark unit test. This will thus cover the new code, without adding specific tests.